### PR TITLE
Added redis NLog target

### PR DESCRIPTION
**Pull request by froberg**
- Removed EventHub NLog target

### CODE REVIEW

**Comment by froberg** [comment]
Remove from provider config:
- EventHubConnectionString
- EventHubName

Add to Provider config:
- LoggingRedisConnnectionString
- LoggingRedisKey

File: src/SFA.DAS.CloudService/ServiceConfiguration.Release.cscfg [hunk 1]
```diff
@@ -5,10 +5,10 @@
     <ConfigurationSettings>
       <Setting name="Microsoft.WindowsAzure.Plugins.Diagnostics.ConnectionString" value="__StorageConnectionString__" />
       <Setting name="EnvironmentName" value="__EnvironmentName__" />
-      <Setting name="StorageConnectionString" value="__StorageConnectionString__" />
+      <Setting name="StorageConnectionString" value="__ConfigurationStorageConnectionString__" />
       <Setting name="ConfigurationStorageConnectionString" value="__ConfigurationStorageConnectionString__" />
```
Comment (mgwilliam) [inline src/SFA.DAS.CloudService/ServiceConfiguration.Release.cscfg]: are you sure __ConfigurationStorageConnectionString__ is the correct value here?

Comment (froberg) [inline src/SFA.DAS.CloudService/ServiceConfiguration.Release.cscfg]: Nope that's wrong.
A copy past miss from **ServiceConfiguration.Cloud.cscfg**
Should I change it in that file as well?

Comment (mgwilliam) [inline src/SFA.DAS.CloudService/ServiceConfiguration.Release.cscfg]: see updated comment

Comment (froberg) [inline src/SFA.DAS.CloudService/ServiceConfiguration.Release.cscfg]: Ok, I will remove that then. 

Are we still using the NuGet **NLog.Extensions.AzureTableStorage**, or can I remove it?

**Pull request merged**